asapdiscovery-dataviz/asapdiscovery/dataviz/html_viz.py

```diff
@@ -81,9 +83,9 @@ def __init__(
         if self.color_method == "subpockets":
             self.logger.info("Mapping interactive view by subpocket dict")
         elif self.color_method == "fitness":
-            if self.target == "MERS-CoV-Mpro":
+            if not target_has_fitness_data(self.target):
                 raise NotImplementedError(
-                    "No viral fitness data available for MERS-CoV-Mpro: set `color_method` to `subpockets`."
+                    "No viral fitness data available for target set `color_method` to `subpockets`."
```

Comment:
small typo (missing colon) that makes this sentence confusing, also why not use the target name here?
```suggestion
                    "No viral fitness data available for {self.target}: set `color_method` to `subpockets`."
```